addons/mass_mailing/tests/test_mailing_internals.py

```diff
@@ -474,6 +477,109 @@ def test_mailing_shortener(self):
                     link_params=link_params,
                 )
 
+    @users('user_marketing')
+    @mute_logger('odoo.addons.mail.models.mail_mail')
+    def test_mailing_launch_flow(self):
+        """ Test that mail should be sent directly if total recipients are less than value
+        specified by config param 'mass_mailing_immediate_send_nbr', otherwise it should
+        enqueue the mailing """
+        res_partners = self._create_partner()
```

Comment:
can you add `5` explicitly ? (will help to understand how many partners are created without scrolling to `_create_partner`)